modules/ballerina-native/src/main/java/org/ballerinalang/nativeimpl/connectors/data/sql/client/UpdateWithGeneratedKeyColumns.java

```diff
@@ -56,8 +57,8 @@
     public BValue execute(Context context) {
         BConnector bConnector = (BConnector) getArgument(context, 0);
         String query = getArgument(context, 1).stringValue();
-        BArray<BString> keyColumns = ((BArray<BString>) getArgument(context, 2));
-        BArray parameters = (BArray) getArgument(context, 3);
+        BArray parameters = (BArray) getArgument(context, 2);
```

Comment:
For all the other SQL Connector actions the parameter order is SQLConnector, SQLQuery and Parameters. This is an overloaded method for the action "updateWithGeneratedKeys" which has one more additional string array parameter for 'keyColumns. To comply with the other actions initial 3 parameters were kept in the same order.